manager-bundle/src/Resources/skeleton/config/parameters.yml

```diff
@@ -1,5 +1,7 @@
 # Default parameters
 parameters:
+    env(APP_SECRET): '%secret%'
+    env(DATABASE_URL): 'mysql://%database_user%:%database_password%@%database_host%:%database_port%/%database_name%'
```

Comment:
If we have a database URL, we do not need the `database_*` parameters anymore, do we?

Reply:
we need them for backwards compatibility and the install tool